src/SettingStore.php

```diff
@@ -48,6 +48,29 @@ public function get($key, $default = null)
 	}
 
 	/**
+	 * Get a flattened array from the settings data.
+	 *
+	 * @param  array   $array   Optional array of settings values, with a fallback to all settings.
```

Comment:
This is a bit confusing. Maybe make an optional `$keys` argument instead? Similar to how `Request::only` works.